packages/react-components/src/templates/NewsAndEventsPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import {
+  perRem,
+  tabletScreen,
+  mobileScreen,
+  vminLinearCalcClamped,
+} from '../pixels';
+import { pearl } from '../colors';
+import NewsAndEventsPageHeader from './NewsAndEventsPageHeader';
+import { contentSidePaddingWithNavigation } from '../layout';
+
+const mainStyles = css({
+  alignSelf: 'stretch',
+  background: pearl.rgb,
+  display: 'grid',
+  gridRowGap: `${vminLinearCalcClamped(
```

Comment:
I wouldn't overdo it with `Clamped`, only use if necessary. I was hesitant to add it because of this, it's more for ensuring things don't go into negative range. Extremely large screen (well beyond large desktop) warrants extreme numbers.

Reply:
We probably want to address this in a separate PR since this is being used on every listing?

Reply:
This was picked up by James in an earlier PR. The issue is that the jump is quite large from mobile to tablet so it was very large at large desktop size.

Reply:
Ohh I'm sorry, thought it was a mobile - large desktop scale